recipe/meta.yaml

```diff
@@ -36,23 +39,29 @@ requirements:
     - pillow >=3.1.1
     - pyct >=0.4.5
     - scipy
-    - toolz >=0.7.4
     - xarray >=0.9.6
+  run_constrained:
+    # 2022/7/22: ImportError: Numba needs NumPy 1.21 or less
+    # numpy has a hard upper limit currently for numba 0.55.1
+    - numpy >=1.18,<1.22
 
 test:
   requires:
     - pip
     - fastparquet >=0.1.6
     - flake8
-    - nbsmoke >=0.2.6
-    - pytest >=3.9.3,<6.0
+    - nbsmoke >=0.5.0
```

Comment:
Noticed they actually pin to `nbsmoke >0.5` (excludes `v0.5.0` specifically?)
https://github.com/holoviz/datashader/blob/v0.14.1/setup.py#L44

Reply:
I'm addressing this https://github.com/AnacondaRecipes/nbsmoke-feedstock/pull/2. I need to upload the artifacts and fix `nbsmoke` pinning

Reply:
`nbsmoke 0.6.0` is available. Updated pinning for it